assets/queries/openAPI/2.0/response_object_incorrect_ref copy/test/positive1.json

```diff
@@ -0,0 +1,62 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Simple API Overview",
+    "version": "1.0.0"
+  },
+  "paths": {
+    "/": {
+      "get": {
+        "operationId": "listVersionsv2",
+        "summary": "List API versions",
+        "responses": {
+          "200": {
+            "$ref": "#/definitions/User"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/limitParam"
+          }
+        ]
+      }
+    }
+  },
+  "responses": {
+    "Success": {
+      "description": "An array with users",
+      "schema": {
+        "$ref": "#/definitions/User"
+      }
+    }
+  },
+  "parameters": {
+    "limitParam": {
+      "name": "limit",
+      "in": "query",
```

Comment:
Here should be `"in": "body"`. In the Parameters Object, the only valid value for `in` (when `schema` is defined) is `body`